R/repair_draws.R

```diff
@@ -140,6 +140,7 @@ repair_chain_ids <- function(chain_ids) {
 #' @param iteration_ids A vector of iteration indices
 #' @noRd
 compute_draw_ids <- function(chain_ids, iteration_ids) {
+  assert_true(length(chain_ids) == length(iteration_ids))
```

Comment:
Without this assertion, there is room for error:
```r
compute_draw_ids(chain_ids = 1:2, iteration_ids = 1:10)
 [1]  1 12  3 14  5 16  7 18  9 20
```

Reply:
Thanks, Good catch!